charts/nri-kube-events/templates/_helpers.tpl

```diff
@@ -141,3 +141,31 @@ licenseKey and cluster are set.
 {{- $customSecretLicenseKey := include "nri-kube-events.customSecretLicenseKey" . -}}
 {{- and (or $licenseKey (and $customSecretName $customSecretLicenseKey)) $cluster}}
 {{- end -}}
+
+{{/*
+Return the cluster name
+*/}}
+{{- define "newrelic.cluster" -}}
+{{- if .Values.cluster -}}
+  {{- .Values.cluster -}}
```

Comment:
Cluster name is populated already in a different way (I do not fully remember how).

I would avoid adding it also as a custom attribute. Therefore I would have merely customAttributesWithoutClusterName.

Reply:
You are right. Here: https://github.com/newrelic/helm-charts/pull/598/files/0ce03b02a64c24836fef4d55cbf3bea37bcaa068#diff-131da0b26df332b77efd10d0695ed21bff8234051410998ae18562a7d0a43b79R74-R87

And here: https://github.com/newrelic/helm-charts/blob/0ce03b02a64c24836fef4d55cbf3bea37bcaa068/charts/nri-kube-events/templates/configmap.yaml#L13

I am removing those changes.